ddapm_test_agent/apmtelemetry.py

```diff
@@ -0,0 +1,12 @@
+import json
+from typing import Any
+from typing import cast
+
+
+TelemetryEvent = dict[str, Any]
```

Comment:
I think this breaks Python <3.9 compatibility (forget specifically which version introduced the type thing)

```suggestion
TelemetryEvent = Dict[str, Any]
```

Reply:
Can we/should we be more specific about the type?

Reply:
yes, I want to in the future, but this should be good enough for now.

we have an openapi spec for this endpoint, so we can/should add better validation